readme.md

```diff
@@ -1727,6 +1727,7 @@ You can even inspect and change the props of components, and see the results imm
 - [ink-text-input](https://github.com/vadimdemedes/ink-text-input) - Text input.
 - [ink-spinner](https://github.com/vadimdemedes/ink-spinner) - Spinner.
 - [ink-select-input](https://github.com/vadimdemedes/ink-select-input) - Select (dropdown) input.
+- [ink-select-input-horizontal](https://github.com/H3RSKO/ink-select-input-horizontal) - Mod of the above componenet adding color and orientation customization.
```

Comment:
"mod" is not a word, and you have a typo.

Reply:
Fixed typo and changed 'mod' to modification. Thought I would be going over line length but apparently not. 

Regarding the original component, we had put this together for our own project which had different default settings, and only after added in the customizability. 

I can work on modifying the original instead. Just give me a few days.